src/pyseekdb/utils/embedding_functions/google_vertex_embedding_function.py

```
@@ -0,0 +1,130 @@
+        try:
+            import vertexai
+            from vertexai.language_models import TextEmbeddingModel
+        except ImportError as error:
+            raise ValueError(
+                "The vertexai python package is not installed. Please install it with `pip install google-cloud-aiplatform`"
+            ) from error
+
+        self.api_key_env = api_key_env
+        self.api_key = os.getenv(api_key_env)
+        if not self.api_key:
+            raise ValueError(f"The {api_key_env} environment variable is not set.")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)